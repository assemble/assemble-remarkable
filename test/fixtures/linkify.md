# should linkify

https://github.com/jonschlinkert/remarkable
