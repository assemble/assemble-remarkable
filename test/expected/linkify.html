<h1>should linkify</h1>
<p><a href="https://github.com/jonschlinkert/remarkable">https://github.com/jonschlinkert/remarkable</a></p>
